src/frontend/pages/Home.jsx

```diff
@@ -0,0 +1,120 @@
+import Card from "../components/Card";
+import Nav from "../components/Nav";
+import coverImage from "../assets/images/cover.jpeg";
+// section collection image
+import bestSellers from "../assets/images/bestsellers.jpeg";
+import newReleases from "../assets/images/new release.jpeg";
+import expertPicks from "../assets/images/expert.jpeg";
+// section genre image
+import contemporaryFiction from "../assets/images/book cluster.jpeg";
+import selfHelp from "../assets/images/self help.jpeg";
+import biographies from "../assets/images/bio.jpeg";
+import spirituality from "../assets/images/spiritual.jpeg";
+import mythology from "../assets/images/history.jpeg";
```

Comment:
Is there any way to import these images from a single source i.e. [barrel import](https://medium.com/@klauskpm/do-a-barrel-export-aa5b79b76b05)? Cause that will increase the readability of the code.

Reply:
Thank you for your suggestion, I have made the changes.